src/Collection.php

```diff
@@ -552,4 +552,14 @@ public function merge($items)
 
         return $col;
     }
+
+    public function first()
+    {
+        return reset($this->items);
```

Comment:
Reset and End will return false if the collection is empty, but that could be an odd response, especially if you have a list of booleans. If the internal array is empty, should there be an exception thrown?